wiki/Game_modifier/Double_Time/ko.md

```diff
@@ -0,0 +1,66 @@
+---
+tags:
+  - double time
+  - double-time
+  - DT
+  - mod
+  - game modifier
+  - 더블 타임
+---
+
+# Double Time (모드)
+
+![DT mod icon](/wiki/shared/mods/DT.png "Double Time (DT) mod icon")
+
+*모든 모드의 목록을 확인하려면 [여기](/wiki/Game_modifier)를 클릭해 주세요.*\
+*관련 모드: [Nightcore](/wiki/Game_modifier/Nightcore)*
+* osu! standard , osu! taiko 등은 mode 이고 nightcore, hidden 등은 mod로 둘 다 '모드'입니다.
+* 따라서 혼동 방지를 위해 mode는 모드+, mod는 모드로 작성하였습니다.
+
+## Double Time 모드
+
+- 축약형: DT
+- 종류: 난이도 상승 모드
+- 점수 배수:
+  - ![][o!s]: 1.12x
+  - ![][o!t]: 1.12x
+  - ![][o!c]: 1.06x
+  - ![][o!m]: 1.00x
+- 기본 단축키: `D`
+- 설명: `빨라아아아아아아아!!`
+- DT를 사용할 수 있는 모드+: ![][o!s] ![][o!t] ![][o!c] ![][o!m]
+- 같은 계열의 모드: [Nightcore](/wiki/Game_modifier/Nightcore)
+
+## 상세 설명
+
+*중요: 이 모드는 노래의 속도를 증가시키지만, 노래의 높낮이를 바꾸지 않습니다.*
+
+**Double Time** 모드는 맵의 속도를 1.5배 증가시킵니다. 또한 노래의 길이를 33% 줄이고 [전체적인 난이도(OD)](/wiki/Beatmapping/Overall_difficulty)를 올리며 [원이 생기고 사라지기까지 걸리는 시간(AR)](/wiki/Beatmapping/Approach_rate)과 [체력(HP)](/wiki/Beatmapping/Health)을 감소시키는 [모드](/wiki/Game_modifier)입니다.
+
+Double Time 모드는 ![][o!s]에서 쉬운 난이도의 맵을 플레이할 때 더 많은 [pp](/wiki/Performance_points)를 얻을 수 있는 가장 좋은 모드로 널리 이용되고 있습니다.
+
+### osu!taiko
+
+[osu!taiko](/wiki/Game_mode/osu!taiko)에서는 난이도가 크게 상승하며 판정이 더욱 엄격해집니다. 따라서 [osu!taiko](/wiki/Game_mode/osu!taiko)에서 가장 어려운 모드로 알려져 있으며, 대부분 이 모드를 이용하지 않습니다.
+
+### osu!catch
+
+In [osu!catch](/wiki/Game_mode/osu!catch), there is no overall difficulty to increase. Thus this mod only increases the BPM and AR. As a result, it is only worth a 1.06x multiplier (as opposed to 1.12x multiplier for every other mod).
+[osu!catch](/wiki/Game_mode/osu!catch)에서는 [전체적인 난이도](/wiki/Beatmapping/Overall_difficulty) 증가가 없습니다. 따라서 이 모드는 BPM과 [AR](/wiki/Beatmapping/Approach_rate) 둘만 바뀌기 때문에 1.06배의 값어치만 하게 됩니다. (다른 모든 모드의 1.12배와는 반대의 경우가 됩니다.)
+
+Although this mod does greatly increase the hyperdash leniency, making fruits require hyperdashes that are near impossible to catch in some cases.
+([osu!catch] 플레이하시는 분이 번역 부탁드립니다 ㅠㅠ)
+
+## 그 외
+
+- Double Time 모드를 활성화하면 `Length`, `BPM`, `Objects` 값이 새 값과 함께 밝은 빨간색으로 표시됩니다.(하단 이미지 첨부)
+  - `Objects` 값은 실제로 변화가 없지만 옅은 빨간색으로 표시됩니다.
+- `CS`, `AR`, 그리고 `HP` 값 옆에 대문자 삼각형이 생기고 값이 조금 상승합니다. (하단 이미지 첨부)
+- Double Time 모드가 비트맵 속도를 2배(200%) 증가시키지 않고 비트맵 속도를 1.5배(150%) 상승시키기 때문에 오인할 수 있습니다.
+
+![Changed values](img/GM_DT.jpg "Screenshot of beatmap values being changed by the Double Time mod")
```

Comment:
tooltip and alt-text have to be translated

Reply:
i checked your whole comment,  i will translate it.